client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -125,12 +127,17 @@ class DomainRegistrationSuggestion extends React.Component {
 			suggestion,
 			translate,
 			pendingCheckSuggestion,
+			siteHasDomainCredit,
 		} = this.props;
 		const { domain_name: domain } = suggestion;
 		const isAdded = hasDomainInCart( cart, domain );
 
 		let buttonContent;
 		let buttonStyles = this.props.buttonStyles;
+		const buttonCopyTest =
+			abtest( 'domainSelectButtonCopy' ) === 'purchase'
+				? translate( 'Purchase', { context: 'verb' } )
+				: translate( 'Select', { context: 'Domain suggestion button' } );
```

Comment:
Hi! I've found a possible matching string that has already been translated **42** times:
`translate( 'Select', { context: 'Domain mapping suggestion button'} )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/domain-select-button-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `button`
- `Domain mapping suggestion button`

 Would it make sense to reuse one of the above?